README.md

```diff
@@ -4,6 +4,10 @@ Altapay has made it much easier for you as merchant/developer to receive secure
 
 ## Change Log
 
+### 1.0.2
+
+- False warnings appear from unnecessary security scans because of the composer.lock file
```

Comment:
Make sure you always explain what has been done, like:

"Fix warning ..."
or 
"Remove warning ..."